### DISCUSSION TITLE
Django. Split sending errors to a separate sentry project (2 DSN)

### DISCUSSION THREAD

**Discussion by ERAalex** [getsentry/sentry-python#3089] [Q&A]
**Django project**, currently using the standard error sending to a single DSN:

```
def init_sentry():  
    sentry_sdk.init(  
        dsn='DSN for MAIN project',  
        integrations=[DjangoIntegration(), PyMongoIntegration()],  
        ignore_errors=[DoesNotExist]  
  )
```

**it is necessary** to allocate CELERY errors (only for CeleryIntegration ) to a separate sentry project (as I understand use a separate DSN)

**is it possible** to create a separate sentry_sdk If sentry_sdk is already running?  Something like addition:

```
celery_sentry_client = sentry_sdk.Client(  
    dsn='DSN for SECOND project',  
    integrations=[  
        CeleryIntegration(),  
    ],  
  
)
```

I searched through various forums, but only outdated information, thanks

**Comment by antonpirker** [comment]
As your Django application server and the Celery workers are different processes this is possible. 

You can [load your DSN from the environment](https://docs.sentry.io/platforms/python/configuration/options/) and when you start your Gunicorn (or Uvicorn, Daphne or whatever server you run you Django app in) you can start it with `SENTRY_DSN=dsn1 gunicorn ...` and when you start your celery workers you do it like this `SENTRY_DSN=dsn2 celery -A app worker ...`. 

Then everything that fails in Django will be sent to `dsn1` and everything that fails in your Celery workers will be sent to `dsn2`.
